wpilibjExamples/build.gradle

```diff
@@ -11,20 +11,6 @@ ext {
 
 apply from: "${rootDir}/shared/opencv.gradle"
 
-test {
-    useJUnitPlatform()
-    systemProperty 'junit.jupiter.extensions.autodetection.enabled', 'true'
-    testLogging {
-        events "failed"
-        exceptionFormat "full"
-    }
-    finalizedBy jacocoTestReport
-}
```

Comment:
With removal of this, are the tests still run?

Reply:
Running the `test` task runs all the example-specific tasks, so yes.